user-guide/Basic_Functionality/Visio/tutorials/Using_Visual_Overview_to_switch_spectrum_presets.md

```diff
@@ -0,0 +1,188 @@
+1. Open the spectrum simulation element and navigate to the `Spectrum Analyzer` page if needed
+
+1. In the `Manual` tab of the spectrum sidebar, change the settings to the desired values. e.g. for a satellite signal this could be around 12GHz center frequency and 200MHz Bandwith
+
+1. In the spectrum sidebar, click the `Presets` tab and click the `Manage` button at the bottom.
+
+1. Click the `New...` button
+
+1. Give the preset a name of your choice, e.g. `Satellite`
+
+1. Enable the `Share preset with other users` checkbox
+
+1. Click the `OK` button
+
+1. Click the `OK` button once more to close the Manage presets window
+
+1. Repeat the above steps for a second signal. E.g. For a cable signal, we could use a center frequency of 260 MHz and a Bandwith of 8 MHz. Give it the name `Cable`
+
+## Step 8: Use session variables to link the presets to buttons
+
+In this step, we will use session variables and placeholders to link buttons to the presets.
+
+1. In DataMiner Cube, Navigate back to the `Visio spectrum presets` view and bring Visio to the front from your taskbar. If you have closed Visio previously, you can re-open it by right-clicking in the view card and choosing `Edit in Visio`from the menu.
+
+1. Resize the shape linked to the spectrum analyzer control, to make some space for buttons
+
+1. Create a small rectangular shape, and add the text "Switch to Satellite"
+
+1. Add a shape data field to this shape, with Label `SetVar` and value `varPreset:Satellite (public)`. In this value, `varPreset`is the name of the session variable and `Satellite (public)` refers to the name of the spectrum preset. The (public) suffix is needed to indicate that this is a shared preset.
+
+1. duplicate this shape and change the text and shape data so it refers to the `Cable` preset
+
+1. In the spectrum component shape, replace the fixed preset reference by a session variable placeholder. The new value becomes `preset=[var:varPreset]`
+
+1. Make sure you have no shapes selected
+
+1. Save your Visio file.
+
+1. Click the buttons and verify that the presets are correctly changed.
+
+## Step 9: Display the current selection
+
+1. Add a number of images in the visio file that represent the setup in your system. You can for example use the [Skylicons](https://skyline.be/skylicons)
+
+1. Draw lines that represent the connections between the components. To enhance the visualization, you can use a different line color for the connections to the image that represents the spectrum analyzer.
+
+1. On the lines connected to the spectrum analyzer, add a shape data with label `Show` and value `<A>-A|Value|[var:varPreset]|=Satellite (public)` for the connection to the satellite dish. Refer to [Extended conditional shape manipulation actions](xref:Extended_conditional_shape_manipulation_actions) for more information on the show condition format
+
+1. Add a shape data with label `Show` and value `<A>-A|Value|[var:varPreset]|=Cable (public)` for the connection to the cable
```

Comment:
for the connection to the 'Cable' preset?  (maybe not so clear)

Reply:
added "image" and a screenshot to make this more clear.